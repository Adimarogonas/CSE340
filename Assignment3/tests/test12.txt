{
    if (< x1 y1) {
        if (= x2 y2) {
            if (> x3 y3) {
                while (!= x4 y4) {
                    while(<> x5 y5) {
                        x1 = + y2 x4;
                    }
                }
            }
        }
    }
}
