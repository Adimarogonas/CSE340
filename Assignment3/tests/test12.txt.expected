x1, y1, x2, y2, x4, y4: ? #
x3, y3: ? #
x5, y5: ? #
